app/controllers/redeemify_codes_controller.rb

```diff
@@ -0,0 +1,21 @@
+class RedeemifyCodesController < ApplicationController
+
+  before_action :set_code, only: [:show]
+
+  def show
+    render json: @code
+  end
+
+  private
+
+  def set_code
+    if current_user
```

Comment:
Sorry.  You're comment and my response got erased by my last commit!
I replied that I am using 'before_action', so can we wait until I do the auth stuff to do a authenticate_user method?